src/Zynga/Framework/IO/Web/V1/ManagerTest.hh

```diff
@@ -0,0 +1,92 @@
+<?hh // strict
+
+namespace Zynga\Framework\IO\Web\V1;
+
+use Zynga\Framework\Testing\TestCase\V2\Base as TestCase;
+use Zynga\Framework\Type\V1\UrlBox;
+
+use Zynga\Framework\IO\Web\V1\Curl\CurlInterface;
+use Zynga\Framework\IO\Web\V1\Curl\CurlRequest;
+use Zynga\Framework\IO\Web\V1\Curl\MockedCurlRequest;
+
+class ManagerTest extends TestCase {
+  
+  <<__Override>>
+  public function tearDown(): void {
+    parent::tearDown();
+    Manager::$useMockCurl = false;
+  }
+  
+  public function testMockedCurlRequestClassObject(): void {
```

Comment:
Not sure if you saw but regarding naming, if you remove the prefix it's hard to tell what exactly this is doing (or intended to do):

```
MockedCurlRequestClassObject
```

It looks like a statement of something and not the case that it is testing for.

What you are doing in this test is verifying that you can create a Mocked Curl Request if the mocking mode is set.  So maybe something like this?

```
testMockedCurlRequestReturnedWhenMockModeEnabled
```